Source/MediaInfo/File__Analyze_Streams_Finish.cpp

```diff
@@ -523,7 +523,7 @@ void File__Analyze::Streams_Finish_StreamOnly_General(size_t StreamPos)
     //Exception
     if (Retrieve(Stream_General, 0, General_Format)==__T("AC-3") && (Retrieve(Stream_General, 0, General_Format_Profile).find(__T("E-AC-3"))==0 || Retrieve(Stream_General, 0, General_Format_AdditionalFeatures).find(__T("Dep"))!=string::npos))
     {
-        Fill(Stream_General, 0, General_Format_Extensions, "ac3 eb3 ec3", Unlimited, true, true); //ec3 added because sauf reference files use it
+        Fill(Stream_General, 0, General_Format_Extensions, "ac3 eb3 ec3", Unlimited, true, true); //ec3 added because some reference files use it

```

Comment:
I'm wondering why `dd+` and `eac3` are in the format or automatic config and not there though.

Reply:
I'll change that in order to automatically get extensions from the automatic config.